tutorials/nservicebus-101/lesson-2/tutorial.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "NServiceBus 101 Lesson 2: Sending a command"
+---
+
+Sending and receiving messages is at the core of any NServiceBus system. Durable messages passed between processes allow reliable communication between those processes, even if one of them is temporarily unavailable. In this lesson we'll show how to send and process a message.
```

Comment:
Technically accurate, maybe? But I'm trying to introduce the lesson on "sending and receiving commands"